sdcm/report_templates/results_base.html

```diff
@@ -7,6 +7,7 @@
         .fbold  { font-weight: bold; }
         .red    { color: red; }
         .green  { color: green; }
+        .tan    { color: tan; }
```

Comment:
What do you think about defining these classes like:

```css
.tan, .test_status.ABORTED { color: tan; }
```

and replacing all `if`'s with one line like following: 

```html
<span class='test_status {{ test_status }}'>{{ test_status }}</span>
```
?

Reply:
agree it would be better, but i fillow same pattern we currently have, if we want to change it, we need to do that with separate PR.